### DISCUSSION TITLE
FR: Asciinema Rendering Support

### DISCUSSION THREAD

**Discussion by nosbig** [TriliumNext/Trilium#7357] [Plugin Ideas]
I was curious about whether or not Trilium supports or could support Asciinema rendering? The raw data is text in a custom format, but the open-source platform uses that, similar to an SVG to display a terminal session as it unfolds.

https://asciinema.org

**Comment by zadam** [comment]
This is too niche to be a built-in feature to be honest. A 3rd party plugin might be possible, though.
